### ISSUE TITLE
No Devices Detected but no prompt from Phone

### ISSUE THREAD

**Issue by phlepper** [joaomgcd/Tasker-Permissions#24]
Similar to issue #14, but I am on Linux (Pop! OS 22.04) and when I plug in my phone, I don't get prompted to debug.

I have a Pixel 4a running Android v13.

I only turned on USB debugging (under Dev Options), and didn't see the other two options you mention in the app (Disable Permission Monitoring or USB Debugging (Security Settings)).

Let me know if there is more information I can provide.

Thanks!
phlepper

**Comment by joaomgcd** [comment]
Those other 2 permissions are just for certain devices, they don't always appear. Does it help if you re-launch the app after your phone is already connected?

**Comment by phlepper** [comment]
No, doesn't make a difference if I launch the app before or after connecting. Also no impact pressing the "Reload" button in the app.

I get a sound from my phone when I plug it in to the USB, but no notification on it whatsoever.  Not sure if there is something wrong with my phone setup.  Would it be useful to try to get it to connect with ADB?  At least then I would know it was connecting...

I also tried a different USB port with no luck.

phlepper

**Comment by phlepper** [comment]
I tried again with ADB and could not get it to work with USB connection (the "List of devices attached" was empty).  I turned on wireless debugging on my phone and tried to connect that way.  I was able to successfully pair the phone through wifi, but when I tried to connect (again with ADB), I get a Connection Refused error.

Don't know if this information is helpful, but I'll keep looking into it on my side.  I'm guessing at this point that it isn't anything to do with your app, but something messed up with my phone settings.

Thanks again!

**Comment by phlepper** [comment]
Okay, in further troubleshooting, I ended up using a different USB cable and that worked!  However, when I connected with the App, it sees the phone now, but when I click on the permissions button, I get the following error:

![Screenshot from 2023-03-01 09-42-28](https://user-images.githubusercontent.com/26470747/222206015-58965dc4-8184-4389-9f68-a51bc49e5b83.png)

In looking into this, I found this [stackoverflow answer](https://stackoverflow.com/questions/53887322/adb-devices-no-permissions-user-in-plugdev-group-are-your-udev-rules-wrong) and after creating the rules file and running 'sudo udevadm control --reload-rules' and restarting the app, instead of seeing the phone listed as "unknown (unknown)", it was listed as "Pixel_4a (sunfish)" and clicking on the permissions button, everything worked!

Thanks again and sorry for the bother when it turned out to be a bad USB cable.

phlepper
